pkg/components/wfbackend/wfbackend_component.go

```diff
@@ -0,0 +1,34 @@
+/*
+Copyright 2023 The Dapr Authors
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+	http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+package wfbackend
+
+import (
+	"github.com/dapr/kit/logger"
+)
+
+func NewWorkflowBackendComp(logger logger.Logger) WorkflowBackend {
```

Comment:
What is the purpose of this method?

Reply:
This function is used when components get initialized for backend component like any other component, for example redis_state which gets initialized from [daprd/components](https://github.com/dapr/dapr/blob/9e2bcdc3d8a64b02466a9274e9259f166eedc95a/cmd/daprd/components/state_redis.go#L24) and implementation is present in [component-contrib](https://github.com/dapr/components-contrib/blob/087a597ec5a06484edd28445a96107fa8811fc02/state/redis/redis.go#L109).

Reason for moving the changes from components-contrib to dapr/dapr was because backend components are not fully fledged components like others and it will be easier to maintain them in dapr/dapr itself.